demo/src/app/components/datepicker/demos/adapter/datepicker-adapter.html

```diff
@@ -24,7 +24,9 @@
           <input class="form-control" placeholder="dd/mm/yyyy"
                  name="d2" #c2="ngModel" [(ngModel)]="model2" ngbDatepicker #d2="ngbDatepicker">
           <div class="input-group-append">
-            <button class="btn btn-outline-secondary calendar" (click)="d2.toggle()" type="button"></button>
+            <button class="btn btn-outline-secondary calendar" (click)="d2.toggle()" type="button">
+              <span class="sr-only" i18n="@@ngb.demo.datepicker.popup.choose">Choose date</span>
```

Comment:
Why did you add i18n to the demo app ?? It seems useless as we don't build it with i18n.

Reply:
Removed i18n from demo.